content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/manually.rst

```diff
@@ -0,0 +1,138 @@
+==================
+Manual reservation
+==================
+
+.. _inventory/reservation_methods/manually:
+
+.. |SO| replace:: :abbr:`SO (Sales Order)`
+
+Unlike the *At Confirmation* reservation method, the *Manually* reservation method does **not**
+reserve products automatically.
+
+Instead, once a sales order (SO) is confirmed, product availability **must** be checked manually,
+and the required quantity **must** be reserved manually.
+
+.. seealso::
+   :doc:`About reservation methods <../reservation_methods>`
+
+Configuration
+=============
+
+To set the reservation method to *Manually*, navigate to :menuselection:`Inventory app -->
+Configuration --> Operations Types`. Then, select the desired :guilabel:`Operation Type` to be
+configured, or create a new one by clicking :guilabel:`New`.
+
+In the :guilabel:`General` tab, locate the :guilabel:`Reservation Method` field, and select
+:guilabel:`Manually`.
+
+.. image:: manually/manually-operations-type.png
+   :align: center
+   :alt: Reservation method field on delivery order operation type form.
+
+.. note::
+   When the :guilabel:`Type of Operation` is changed to *Receipt* on an :guilabel:`Operations Type`
+   form, reservation methods are **not** available.
```

Comment:
If "Receipt" is a drop-down menu option please use a :guilabel:, if it is user input please use literal (\`Receipt\`) formatting :)
```suggestion
   When the :guilabel:`Type of Operation` is changed to :guilabel:`Receipt` on an
   :guilabel:`Operations Type` form, reservation methods are **not** available.
```

Reply:
Suggestion implemented ✅